src/components/Table/_table.scss

```diff
@@ -1 +1,23 @@
 @import 'TableHead/table-head';
+@import '~carbon-components/scss/components/data-table/data-table';
+
+.#{$prefix}--toast {
+  margin-bottom: 0.5rem;
+  min-width: 30rem;
+  position: fixed;
+  top: 0;
+  right: 0;
+  z-index: 1;
+}
+
+.#{$prefix}--toolbar-button {
```

Comment:
This should be scoped to the table via a class applied in edit mode, otherwise this is a globally applied style.

Values, colors, etc should be tokens